doc/_templates/index.html

```diff
@@ -11,230 +11,205 @@
-OneHotEncoder but robust to typos or non-normalized categories. <a
-class="reference internal" href="auto_examples/02_feature_interpretation_with_gapencoder.html#example-gap-encoder"><span class="std std-ref">Example</span></a></p>
+
+<hr class="thin-line">
+
+  <!-- Feature Section 2 -->
+<div class="container my-0">
+  <div class="row mb-0 flex-md-row-reverse">
+    <div class="col-md-5">
+      <h4 class="feature-title">Powerful Feature Engineering</h4>
+      <p class="feature-text">Encode text and high cardinality categorical data with the
+        <a class="reference internal"
+        href="reference/generated/skrub.GapEncoder.html#gapencoder"
+        title="skrub.GapEncoder"><code class="xref py py-class docutils
+        literal notranslate"><span class="pre">GapEncoder</span></code></a>
+        and
+        <a class="reference internal"
+        href="reference/generated/skrub.MinHashEncoder.html#minhashencoder"
+        title="skrub.MinHashEncoder"><code class="xref py py-class docutils
+        literal notranslate"><span class="pre">MinHashEncoder</span></code></a>
+        , and extract features from dates with the
+        <a class="reference internal"
+        href="reference/generated/skrub.DatetimeEncoder.html#datetimeencoder"
+        title="skrub.DatetimeEncoder"><code class="xref py py-class docutils
+        literal notranslate"><span class="pre">DatetimeEncoder</span></code></a>
+        .
+      </p>
+      <br>
+      {% include "demo_gap_encoder.html" %}
+    </div>
+    <div class="col-md-7 pe-5">
+      <img src="_static/gap.png">
     </div>
   </div>
 </div>
-<div class="col-lg-4">
-  <div class="sd-card sd-shadow-sm card-landing">
-    <div class="card-body">
-      <h4 class="card-title"><a href="{{ pathto('cleaning') }}">Cleaning</a></h4><p><a class="reference
-internal" href="reference/generated/skrub.deduplicate.html#skrub.deduplicate"
-title="skrub.deduplicate"><code class="xref py py-func docutils literal
-notranslate"><span class="pre">deduplicate()</span></code></a>: merge
-categories of similar morphology (spelling). <a class="reference internal" href="auto_examples/05_deduplication.html"><span class="std std-ref">Example</span></a></p>
-        </div>
-      </div>
+
+<hr class="thin-line">
+
+  <!-- Feature Section 3 -->
+<div class="container my-0">
+  <div class="row mb-0">
+    <div class="col-md-5">
+      <h4 class="feature-title">Interactive Data Exploration</h4>
+      <p class="feature-text">Explore your dataframes interactively with
+        <a class="reference internal"
+        href="reference/generated/skrub.TableReport.html#tablereport"
+        title="skrub.TableReport"><code class="xref py py-class docutils
+        literal notranslate"><span class="pre">TableReport</span></code></a>
+        .</p>
+      <br>
+      {% include "demo_table_report_code.html" %}
+    </div>
+    <div class="col-md-7">
+      <p class="click-table-report-hint">Click anywhere on the table!</p>
```

Comment:
I don't really have a better suggestion but with "click on the table" people might look only at the table and miss the tabs and the column filter... maybe something like "the displayed report is interactive"?... I can't come up with a better wording and it's not a very big deal

Reply:
How about moving the text back to the left panel (for better usage of the space), and writing "Try it out: click anywhere ➔"